packages/flutter_tools/test/commands.shard/hermetic/shell_completion_test.dart

```diff
@@ -22,7 +22,7 @@ void main() {
 
     setUp(() {
       Cache.disableLocking();
-      mockStdio = MockStdio();
+      mockStdio = MockStdio()..stdout.terminalColumns = 80;
```

Comment:
Once we use your new matcher, will we need to set this, or should it all just work?

Reply:
This should prevent "unspecified mock value" errors. Seems to have different origin.

Reply:
Ahh yes, you're right.